control-plane/src/config_server.rs

```diff
@@ -418,13 +426,17 @@ async fn get_acme_jwk(acme_account_details: AcmeAccountDetails) -> ServerResult<
         .map_err(ServerError::HyperHttp)
 }
 
-fn validate_order_identifiers(
+fn valid_order_identifiers(
     payload: NewOrderPayload,
     cage_context: configuration::CageContext,
 ) -> bool {
+    let underscored_app_uuid = cage_context.app_uuid.replace('-', "_");
```

Comment:
can use the context function here:
```suggestion
    let underscored_app_uuid = cage_context.underscored_app_uuid();
```